packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -87,6 +100,9 @@ foreach ($directory in $directories) {
 
     $generationDir = $cadlRanchRoot
     foreach ($folder in $folders) {
+        if ($folder -eq "specs") {
+            $folder = "http" # Keep consistent with the previous folder name
+        }
```

Comment:
could you create an issue to track this?
We should change remove this once this PR is merged.

Reply:
I don't plan to remove. I think `http` is more reasonable than `specs`.

Reply:
Then please change the comment here.